apps/web/src/server/views/_includes/components/InfoBox/InfoBox.njk

```diff
@@ -0,0 +1,7 @@
+{% macro InfoBox(message) %}
+
+	<div class="pi-info-box govuk-table govuk-!-padding-9">
```

Comment:
Not sure why you used `govuk-table` but this is not a table and it just makes the code confusing, just manually add the code you need.

Reply:
👍